bom/build.gradle.kts

```diff
@@ -111,6 +111,7 @@ dependencies {
         apiv("org.apache.commons:commons-pool2")
         apiv("org.apache.commons:commons-collections4")
         apiv("org.apache.commons:commons-text")
+        apiv("org.apache.flink:flink-table-code-splitter")
```

Comment:
Add a note above this list of declarations that we import flink-table-code-splitter but we must not import core flink modules.

Reply:
Done.